src/components/menu.js

```diff
@@ -3,6 +3,18 @@ import classnames from "classnames";
 
 import styles from "./menu.scss";
 
+const registerToggleActive = (add, toggleActive) => {
```

Comment:
Look, my intuition is not in favor of this solution.
Some of my concerns are related to the fact that we're adding an event listeners to areas that we don't own and on top of that we stopPropagation due to this event.
Additionally, we're assuming iframes are not added/removed during the lifetime of the application which is an assumption that might not be correct (e.g. a dialog with an iframe inside).
I'd love to hear a 2nd opinion on this.

Reply:
Thanks @liorheber 

So @yarin-kenshoo and I were discussing your comment and thought it would be safer to just send an iframe id as a prop, does it sound better?

Also we (and the product) are aware it then will not work with different iframes, only the one delivered in the prop

Reply:
Tested and it works - ready to merge.